### DISCUSSION TITLE
Comando "dotnet run" só gera HTTP

### DISCUSSION THREAD

**Discussion by VinixGonzalez** [balta-io/forum#262] [Tira Dúvidas] [answered]
Boa tarde,

na aula 3,  Protocolo HTTP, DNS e fundamentos, ao rodar o comando "dotnet run", só é gerada a url com HTTP e não HTTPS, como consigo rodar em HTTPS ?

![image](https://user-images.githubusercontent.com/47197477/218265989-399f661f-d65e-4597-a707-90d955c46af7.png)

obrigado!

**Comment by Gilthayllor** [comment, marked as answer]
Boa tarde!

Veja se sua classe Program.cs possui uma chamada para `app.UseHttpsRedirection();`

Se não possuir, adicione ela após o `app.Build()`.

No arquivo `launchSettings.json` procure pela propriedade `"applicationUrl"` e adicione uma porta para o HTTPS.
Exemplo: `"applicationUrl": "https://localhost:5001;http://localhost:5000"`

Espero que isto lhe ajude.

Abraços!
